packages/fx-core/src/common/local/localStateHelper.ts

```diff
@@ -91,6 +91,9 @@ export async function convertToLocalEnvs(
     localEnvs[LocalEnvFrontendKeys.Https] = frontendConfigs?.get(
       LocalStateFrontendKeys.Https
     ) as string;
+    if (!localEnvs[LocalEnvFrontendKeys.Https]) {
```

Comment:
In what case will `https` be `undefined`?

Reply:
In some samples, like ```hello world tab withbackend```. Normally this will not happen.